models/release.go

```diff
@@ -217,6 +223,28 @@ func GetReleasesByRepoID(repoID int64, opts FindReleasesOptions) ([]*Release, er
 	return rels, sess.Find(&rels)
 }
 
+// GetLatestReleaseByRepoID returns the latest release for a repository
+func GetLatestReleaseByRepoID(repoID int64) (*Release, error) {
+	cond := builder.NewCond().
+		And(builder.Eq{"repo_id": repoID}).
```

Comment:
In fact, you can use `builer.Eq{
   "repo_id": repoID,
   "is_draft": false,
   ...
}`

Reply:
Good to know